docs/en/3_model_zoo.md

```diff
@@ -5,21 +5,21 @@
 - Number of papers: 46
   - ALGORITHM: 47
 - Tasks:
-  - image2image
-  - unconditional gans
-  - video super-resolution
   - conditional gans
-  - inpainting
+  - image2image translation
```

Comment:
maybe we can just keep `image2image` and remove `image2image translation`

Reply:
this markdown file is generated automatically. I will fix the generation code in another PR.